### ISSUE TITLE
Виза 

### ISSUE THREAD

**Issue by Davibiyuu** [Davibiyuu/zashitacoil#280]
Подскажите мне какие документы необходимы для визы невесты в Великобританию

**Comment by diorjio** [comment]
Да, действительно, очень важно от и до изучить все для того, чтобы не совершить ошибок, чтобы все прошло действительно гладко. От себя советую вот здесь изучить все на тему того, какие документы необходимы, все про визу жены, визу невесты Великобритании https://smolensk-i.ru/partners/viza-zheny-i-viza-nevesty-v-velikobritaniyu-kak-poluchit-pmzh-i-grazhdanstvo_337078 , как получить ПМЖ и гражданство, какие необходимо выполнить требования, какие документы нужны, однозначно рекомендую, почитайте, это даст ответы на многие вопросы
